admin-dev/themes/new-theme/js/product-page/combination.js

```diff
@@ -8,7 +8,7 @@ export default function() {
     let idsProductAttribute = $jsCombinationsList.data('ids-product-attribute').toString().split(',');
     let idsCount = idsProductAttribute.length;
     let currentCount = 0;
-    let step = 5;
+    let step = 50;
```

Comment:
Hmm, I smell the memory allocation exception by loading 50 of them in a same row.

Reply:
I agree, needs a memory benchmark before it can be increased

Reply:
of course the memory increase, but in our limits:

![call_combination](https://user-images.githubusercontent.com/1247388/37658407-bc7daec2-2c4d-11e8-9011-0000bbcab5cb.png)

Reply:
Ok for me!